bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/AudioFormat.java

```diff
@@ -457,13 +457,13 @@ public boolean equals(@Nullable Object obj) {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
-        result = prime * result + ((bigEndian == null) ? 0 : bigEndian.hashCode());
```

Comment:
Maybe you could use ```Object```, it should already include ```hashCode``` (and is might be overwritten in the derived objects, still works as expected).
```Object``` would still work if someone changes the datatypes above. Just imagine someone to change Integer to String - then we would fail silently (drop to default value "0").